Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -272,15 +269,20 @@ define([
 
         var batchTableResources = new Cesium3DTileBatchTableResources(this, instancesLength);
         this.batchTableResources = batchTableResources;
-        if (batchTableJSONByteLength > 0) {
-            var batchTableString = getStringFromTypedArray(uint8Array, byteOffset, batchTableJSONByteLength);
-            batchTableResources.batchTable = JSON.parse(batchTableString);
-            byteOffset += batchTableJSONByteLength;
+        if (batchTableJsonByteLength > 0) {
```

Comment:
Perhaps there should be a helper function for parsing this part of the header (and maybe for the feature table properties as well?).  Could be a static `from` function on `Cesium3DTileBatchTableResources` or the constructor.